_datademos/cbio_how_to_get_genie_on_fh_cbio.md

```diff
@@ -0,0 +1,62 @@
+---
+title: How to Request Access to GENIE Data (Fred Hutch cBioPortal)
+last_modified_at: 2025-04-18
+main_authors: sitapriyamoorthi, tlambert
+primary_reviewers: tfirman, laderast  
+---
+
+## 1. Intake
```

Comment:
@sitapriyamoorthi let's remove the numbering to keep consistent formatting across the Wiki